com.github.IsmaelMartinez.teams_for_linux.yml

```diff
@@ -22,6 +22,8 @@ finish-args:
   - --talk-name=org.gtk.Notifications
   - --talk-name=org.gnome.SessionManager
   - --env=XDG_CURRENT_DESKTOP=Unity
+  # required to fix cursor scaling on wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
```

Comment:
@bbhtt would you mind reviewing this one?

Reply:
Yes, LGTM this is a known issue, see https://github.com/flathub/org.electronjs.Electron2.BaseApp/issues/42

Reply:
Thanks!